wagtail/wagtailimages/models.py

```diff
@@ -458,7 +458,8 @@ def _search_for_operations(cls):
 
 
 class AbstractRendition(models.Model):
-    filter = models.ForeignKey(Filter, related_name='+')
+    filter = models.ForeignKey(Filter, related_name='+', null=True, blank=True)
```

Comment:
It means that when the field is dropped in the next release, users with custom rendition models won't be prompted for a default value for the reverse migration when they run makemigrations.

(There _may_ have been more to it, but I think that's the gist of it...)